contracts/nois-proxy/src/msg.rs

```diff
@@ -20,14 +21,26 @@ pub struct InstantiateMsg {
 #[cw_serde]
 pub enum ExecuteMsg {
     // KEEP IN SYNC WITH ProxyExecuteMsg::GetNextRandomness
-    GetNextRandomness { job_id: String },
+    GetNextRandomness {
+        job_id: String,
+    },
     // KEEP IN SYNC WITH ProxyExecuteMsg::GetRandomnessAfter
-    GetRandomnessAfter { after: Timestamp, job_id: String },
-
+    GetRandomnessAfter {
+        after: Timestamp,
+        job_id: String,
+    },
+    /// Set the config
+    SetConfig {
+        new_config: NewConfig,
```

Comment:
Let's better inline all the arguments here. You can silence all the clippy warnings.

Reply:
You espelcially avoid the nesting in the JSON you send to the contact, which is now

```json
{"set_sonfig": { "new_config": { "myarg": 1 } }}
```